config/Migrations/20240302162526_TaggedMessages.php

```diff
@@ -0,0 +1,92 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class TaggedMessages extends AbstractMigration
+{
+    /**
+     * Up Method.
+     *
+     * More information on this method is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-up-method
+     *
+     * @return void
+     */
+    public function up(): void
+    {
+        $this->table('tagged_messages', ['id' => false, 'primary_key' => ['id']])
+            ->addColumn('id', 'uuid', [
+                'default' => null,
+                'limit' => null,
+                'null' => false,
+            ])
+            ->addColumn('message', 'string', [
+                'default' => null,
+                'limit' => 4096,
+                'null' => false,
+            ])
+            ->addColumn('permalink', 'string', [
+                'default' => null,
+                'limit' => 255,
+                'null' => false,
+            ])
```

Comment:
I would move the message column into the `messages` table and reference it here via a `message_id` column.